logging_metrics_performance/enterprise_logging/README.md

```diff
@@ -0,0 +1,36 @@
+Currently there are two ways we can execute the test harness
+
+
+1. Run the script which gathers pbench metrics for a chosen test and for a given set of cluster nodes.
+
+	Ex:
+
+	  # this will execute the E2E's with a scale option of 5, while gathering metrics with pbench.
+
+	  ./perftest.sh -n LoggingSoak_E2E_5 -e 5
+
+	  ./perftest.sh -h
+
+2. Run the scripts individually from the 'utils' folder. This will not gather metrics with pbench.
+
+	Ex:
+	  1) ./utils/jdspammer.sh -r 60 -l 512
+
+	  2) EXPORT FD=100 ; EXPORT ES=10 ; ./utils/fluentd_autoscaler.sh
+
+
+
+E2Es:
+
+Relevant Until https://github.com/kubernetes/kubernetes/pull/24536/files merges into upstream kube and openshift...
+
+To run the e2es, you essentially follow these steps.
+
+- git clone jayunit100/kubernetes
+- cd kubernetes and checkout branch "LoggingSoak"
```

Comment:
Could you mark as a TODO to change instructions once this lands in a rebase to just run from extended.